openhantek/src/configdialog/DsoConfigProbePage.h

```diff
@@ -0,0 +1,47 @@
+//
+// Created by jacopo on 08/01/18.
+//
+
+#ifndef OPENHANTEKPROJECT_DSOCONFIGPROBEPAGE_H
+#define OPENHANTEKPROJECT_DSOCONFIGPROBEPAGE_H
```

Comment:
`#pragma once` is used in this project, no header guards

Reply:
I'll remove them! Just an habit.